app/utils/camera_discovery.py

```
@@ -170,6 +170,125 @@ def _onvif_get_device_info(xaddr: str, timeout: int = 2) -> dict[str, str]:
     return info
 
 
+def autodetect_onvif_endpoints(url: str, timeout: int = 3) -> dict[str, str]:
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)